examples/angular/src/pages/ui/components/authenticator/sign-up-with-email/sign-up-with-email.component.ts

```diff
@@ -10,4 +10,20 @@ export class SignUpWithEmailComponent {
   constructor() {
     Amplify.configure(awsExports);
   }
+
+  services = {
+    async handleSignUp(formData: Record<string, any>) {
```

Comment:
Is there a better Type or Interface that customers can depend on for this `formData`?

We know that this will be something like `LoginMechanism | SignUpAttributes`.  Or because this can be _anything_, they should use `Record`?

Reply:
Let me think about this, maybe we can.